lib/Raven/Breadcrumbs.php

```diff
@@ -36,7 +33,7 @@ public function reset()
     {
         $this->count = 0;
         $this->pos = 0;
-        $this->buffer = array();
```

Comment:
I reverted back the changes to let you handle code-style issues in your PR. The changes of this PR only relate to add namespaces where necessary to switch to PSR-4 standard. I still used in some places (mainly the one where I add to change the code to make it working with PSR-4) the `::class` notation introduced with PHP 5.5, so I did not revert the change in Travis that does not test anymore those versions, but I updated `composer.json` accordingly to allow it.